<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ControlPanelWidget" name="controlWidget" native="true"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabPage">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="worlTabsWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="initializatioPage">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="initializationLabel">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Initialization...</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionLists"/>
    <addaction name="separator"/>
    <addaction name="actionSolverLogs"/>
    <addaction name="actionSolverErrors"/>
    <addaction name="actionSolverSetup"/>
    <addaction name="separator"/>
    <addaction name="actionLocationSetup"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionRestartServer"/>
    <addaction name="actionConnections"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAtlantSetup">
   <property name="text">
    <string>Atlant setup</string>
   </property>
  </action>
  <action name="actionLists">
   <property name="text">
    <string>Lists</string>
   </property>
  </action>
  <action name="actionSolverLogs">
   <property name="text">
    <string>SolverLogs</string>
   </property>
  </action>
  <action name="actionRestartServer">
   <property name="text">
    <string>Restart Server</string>
   </property>
  </action>
  <action name="actionSolverErrors">
   <property name="text">
    <string>SolverErrors</string>
   </property>
  </action>
  <action name="actionSolverSetup">
   <property name="text">
    <string>SolverSetup</string>
   </property>
  </action>
  <action name="actionLocationSetup">
   <property name="text">
    <string>LocationSetup</string>
   </property>
  </action>
  <action name="actionConnections">
   <property name="text">
    <string>Connections</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControlPanelWidget</class>
   <extends>QWidget</extends>
   <header>ControlPanel/ControlPanelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
